core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -12,6 +13,7 @@
 
     private Logger(java.util.logging.Logger logger) {
         this.jul = logger;
+        new LoggerConfigurator(jul);
```

Comment:
This looks quite strange. On one side we create an instance of `LoggerConfigurator` but then refer to it through global state / statics?